### PR TITLE
Crab 17 now automatically goes away after a while, but has stronger defense

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd / Qustinnus
 tweak: Crab 17 now goes away after a while, but is more fortified.
/:cl:

The intended gameplay of this machine was that brute-force would not let you cancel the effects, but that it was a forced trip to wherever the machine is which lets the traitor use it as a time to do what he needs to do, instead now people just attack it and destroy it in around 30~ seconds. Now the machine automatically goes away, but people still have to check out if they dont want to lose their funds.

### CODE REVIEW

File: code/game/objects/items/crab17.dm [hunk 1]
```diff
@@ -172,6 +172,7 @@
 			account.transfer_money(B, amount)
 			B.bank_card_talk("You have lost [percentage_lost * 100]% of your funds! A spacecoin credit deposit machine is located at: [get_area(src)].")
 	addtimer(CALLBACK(src, .proc/dump), 150) //Drain every 15 seconds
+	addtimer(CALLBACK(src, .proc/qdel, src), 8 MINUTES) //Drain every 15 seconds
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/crab17.dm]: `//Drain every 15 seconds`
hmmmmmm

Comment (Qustinnus) [reply code/game/objects/items/crab17.dm]: what?

Comment (Qustinnus) [reply code/game/objects/items/crab17.dm]: oh

**Comment by ExcessiveUseOfCobblestone** [comment]
5000 is x2 megafauna who prob don't have the generous armor values of this guy.

**Comment by PKPenguin321** [comment]
yeah this is basically making it a "click 3000 times on this thing"

**Comment by Qustinnus** [comment]
you're not supposed to destroy it thats the point 🤔

**Comment by Qustinnus** [comment]
you realize you can just use your ID on it to stop it from draining, right

**Pull request merged**